src/Moq/ISetup.cs

```diff
@@ -14,5 +14,15 @@ public interface ISetup
 		///   The setup expression.
 		/// </summary>
 		LambdaExpression Expression { get; }
+
+		/// <summary>
+		///   Gets whether this setup is conditional.
+		/// </summary>
+		bool IsConditional { get; }
+
+		/// <summary>
+		///   Gets whether this setup is disabled.
+		/// </summary>
+		bool IsDisabled { get; }
```

Comment:
The reason for renaming this to `IsOverridden` in a later commit is that the new name better maintains the abstraction: Noone should have to know (outside of the Moq code base) that overridden setups are implemented using some kind of "disabled" flag.

(Also, it will be much more likely that users will request a `void Disable()` method once they see a `IsDisabled` property, which would be handy (essentially giving users the means to "remove" single setups) but hard to implement properly at this time. Imagine disabling a setup that overrides an earlier setup with the same expression; the latter would then have to be automatically re-enabled. Given today's implementation, that would be fairly tricky to achieve efficiently. Let's solve that problem at a later time.)